source/css/_schemes/Gemini/index.styl

```diff
@@ -76,8 +76,7 @@ use_seo = hexo-config('seo');
 }
 
 // Post delimiters.
-.post-eof,
-.post-spread {
```

Comment:
Are we remove this class? I don't remember...

Reply:
I didn't find `post-spread` in NexT... seems it has been removed

Reply:
Here: https://github.com/theme-next/hexo-theme-next/pull/357/files. `.post-spread` no longer exist